src/coreclr/System.Private.CoreLib/src/System/MulticastDelegate.cs

```diff
@@ -25,13 +25,15 @@ public abstract class MulticastDelegate : Delegate
         // This constructor is called from the class generated by the
         //    compiler generated code (This must match the constructor
         //    in Delegate
+        [RequiresUnreferencedCode("The target method might be removed")]
         protected MulticastDelegate(object target, string method) : base(target, method)
         {
         }
 
         // This constructor is called from a class to generate a
         // delegate based upon a static method name and the Type object
         // for the class defining the method.
+        [RequiresUnreferencedCode("The target method might be removed")]
```

Comment:
Same comment as on `Delegate` should be possible to fix this with `DynamicallyAccessedMembers`.